src/pages/faq.tsx

```diff
@@ -1,104 +1,52 @@
-    {stateAction[0][0] ? (
-    <div className="pl-[20px] h-[140px] sm:h-[100px] bg-gray bg-opacity-50 border-[1px] border-cerise flex items-center ">
-    <p className="text-white">{t.faq.table3text1_1} <a href={t.url.forCompany} target="blank" className="underline text-cerise">  {t.url.forCompany} </a>{t.faq.table3text1_2} <a className="text-cerise" href="mailto:sales@ddagen.se">sales@ddagen.se</a></p>
-    </div>) : null}
-
-    <button onClick={() => stateAction[1][1](!stateAction[1][0])} id="accordion" className="pl-[20px] text-left text-white h-[85px] sm:h-[55px] w-full bg-slate-50 bg-opacity-20 border-[3px] border-cerise">{t.faq.table3row2}</button>
-    {stateAction[1][0] ? (
-    <div className="pl-[20px] h-[260px] sm:h-[150px] bg-gray bg-opacity-50 border-[1px] border-cerise flex items-center">
-    <p className="text-white">{t.faq.table3text2_1} <a href={t.url.companyForm} target="blank" className="underline text-cerise"> {t.url.companyForm}</a></p>
-    </div>) : null}
-
-    <button onClick={() => stateAction[2][1](!stateAction[2][0])} id="accordion" className="pl-[20px] text-left text-white h-[55px] w-full bg-slate-50 bg-opacity-20 border-[3px] border-cerise">{t.faq.table3row3}</button>
-    {stateAction[2][0] ? (
-    <div className="pl-[20px] h-[100px] bg-gray bg-opacity-50 border-[1px] border-cerise flex items-center">
-    <p className="text-white">{t.faq.table3text3} <a className="text-cerise" href="mailto:ansvarig@ddagen.se">ansvarig@ddagen.se</a></p>
-    </div>) : null}
-
-    <button onClick={() => stateAction[3][1](!stateAction[3][0])} id="accordion" className="pl-[20px] text-left text-white h-[85px] sm:h-[55px] w-full bg-slate-50 bg-opacity-20 border-[3px] border-cerise">{t.faq.table3row4}</button>
-    {stateAction[3][0] ? (
-    <div className="pl-[20px] h-[110px] sm:h-[100px] bg-gray bg-opacity-50 border-[1px] border-cerise flex items-center ">
-    <p className="text-white">{t.faq.table3text4} <a className="text-cerise" href="mailto:sales@ddagen.se">sales@ddagen.se</a></p>
-    </div>) : null}
-</div>)
-}
-
-function Table4(stateAction: Array<[boolean, React.Dispatch<React.SetStateAction<boolean>>]>){
-  const t = useLocale()
-  return(
-  <div className="mt-[50px]">
-
-    <button onClick={() => stateAction[0][1](!stateAction[0][0])} id="accordion" className="pl-[20px] text-left text-white h-[130px] sm:h-[55px] w-full bg-slate-50 bg-opacity-20 border-[3px] border-cerise"> {t.faq.table4row1}</button>
-    {stateAction[0][0] ? (
-    <div className="pl-[20px] h-[350px] sm:h-[100px] bg-gray bg-opacity-50 border-[1px] border-cerise flex items-center ">
-    <p className="text-white">{t.faq.table4text1} <a className="text-cerise" href="mailto:alexandre.moch@ddagen.se">alexandre.moch@ddagen.se</a></p>
-    </div>) : null}
+    <div className="mt-[50px]">
+
+    {questions.map((_, i) => (
+      <div>
```

Comment:
We get the following warning right now: 

```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `Faq`. See https://reactjs.org/link/warning-keys for more information.
    at div
    at Faq (webpack-internal:///./src/pages/faq.tsx:94:66)
    at div
    at App (webpack-internal:///./src/pages/_app.tsx:19:11)
    at PathnameContextProviderAdapter (webpack-internal:///./node_modules/next/dist/shared/lib/router/adapters.js:62:11)
    at ErrorBoundary (webpack-internal:///./node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:301:63)
    at ReactDevOverlay (webpack-internal:///./node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:850:919)
    at Container (webpack-internal:///./node_modules/next/dist/client/index.js:62:1)
    at AppContainer (webpack-internal:///./node_modules/next/dist/client/index.js:197:11)
    at Root (webpack-internal:///./node_modules/next/dist/client/index.js:372:11)
```

To fix this we can simply add `key={i}` in this div.

Reply:
Nice, I will fix this.

Reply:
Fixed